### DISCUSSION TITLE
💡 Feature Request: Remove X button when in dungeon and put O in its' place, and remove all the dark, unclickable buttons.

### DISCUSSION THREAD

**Discussion by Dx8pi** [DiscordDungeons/Bugs#1328] [Ideas]
### Is your feature request related to a problem?

X doesn't do anything, only time it is used is when you ignore a chest, in which case the bot will just "re-interact" with the tile, the way you really ignore it is just keep walking over it. So pressing the X doesn't actually do anything useful, it just adds more workload to the bot. I propose putting the O button there instead, which will give you a 2x3 grid of buttons that you can use, this will also allow you to remove the extra dark and unclickable buttons. So instead of having 10 buttons, you'd be down to 6, where all 6 of them are clickable.

So instead of:

O ✓ ^ X []
[] <  v > []

It'll be:

✓ ^ O
< v  >

**Comment by FelixArgyle** [comment]
https://discord.com/channels/172382467385196544/971840667305246731 -- thread has changes for better interaction changes towards dungeons since this was spoke / planned a little while ago to change it entirely to custom buttons (well just the emotes since i smort)
